### PR TITLE
correct auth wording in docstrings

### PR DESCRIPTION
**Pull request by nicholasmhughes**
### What does this PR do?
removes some references to pillar, add references to acct, and corrects some wording.

### Commits signed with GPG?
Yes

Please review [idem-azurerm's Contributing Guide](https://github.com/eitrtechnologies/idem-azurerm/blob/master/.github/CONTRIBUTING.md) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: idem_azurerm/exec/azurerm/authorization/permission.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 :maintainer: <devops@eitr.tech>
 :configuration: This module requires Azure Resource Manager credentials to be passed as keyword arguments
-to every function in order to work properly.
+    to every function in order to work properly.
```
Comment (BellaNotte1986) [inline idem_azurerm/exec/azurerm/authorization/permission.py:9]: Should this be "to every function or via acct in order to work properly." like the other files?

Comment (nicholasmhughes) [reply idem_azurerm/exec/azurerm/authorization/permission.py]: yup. u r smrt

File: idem_azurerm/states/azurerm/postgresql/server.py [hunk 2]
```diff
@@ -232,7 +234,7 @@ async def present(
         if login:
             ret["changes"]["new"]["administrator_login"] = login
         if login_password:
-            ret["changes"]["new"]["administrator_login_password"] = "REDACTED"
+            ret["changes"]["new"]["administrator_login_password"] = "REDACTED"  # nosec
```
Comment (BellaNotte1986) [inline idem_azurerm/states/azurerm/postgresql/server.py:237]: Is this necessary?

Comment (nicholasmhughes) [reply idem_azurerm/states/azurerm/postgresql/server.py:237]: Yup. That tells bandit to stop complaining about a hard-coded password... because it's not.

**Pull request merged**